ansible/roles_ocp_workloads/ocp4_workload_example/readme.adoc

```diff
@@ -22,25 +22,23 @@
 *** This role doesn't do anything here
 *** Debug task will print out: `remove_workload Tasks completed successfully.`
 
-== Review the defaults variable file
+== The defaults variable file
 
 * This file link:./defaults/main.yml[./defaults/main.yml] contains all the variables you need to define to control the deployment of your workload.
 * The variable *ocp_username* is mandatory to assign the workload to the correct OpenShift user.
 * A variable *silent=True* can be passed to suppress debug messages.
 * You can modify any of these default values by adding `-e "variable_name=variable_value"` to the command line
-* Add a dictionary for workload parameters.
+* Add long-name scoped workload parameters. Example: `ocp4_workload_example_machineconfigpool_name: worker`
 
 == Understand how paremeters to your workload should be specified
 
 === Overview
 
-A `dictionary` approach should be used in order to neatly encapsulate parameters to roles.
-
-The role itself defines a dictionary `<role name>_defaults` in the file link:./defaults/main.yml[./defaults/main.yml]. Because this example role is `ocp-workload-example` the dictionary is called `ocp_workload_example_defaults`. Note that `_` is used instead of `-` in the dictionary name to create a valid Ansible dictionary name.
-
-When deploying a workload you can specify another dictionary with the specific parameters for this deployment. This dictionary should be called `<role name>_vars`. In our example this dictionary would be called `ocp_workload_example_vars`.
-
-
+Please use long-name scope parameters for your workload.  
```

Comment:
I'd rephrase to "You *must* use long-name scope parameters for your workload to avoid variable clashing"